cmd/geth/accountcmd.go

```diff
@@ -222,6 +223,74 @@ func accountList(ctx *cli.Context) error {
 	return nil
 }
 
+// tries unlocking the specified account a few times.
+func unlockAccount(ks *keystore.KeyStore, address string, i int, passwords []string) (accounts.Account, string) {
```

Comment:
This seems to be the exact same function we already have in `internal/cli/server/config.go`
Any reason why we're not exporting that to a common util but duplicating the code instead?
Is it just because we have our own cli here and want to reduce upstream efforts?